app/stac_api/models.py

```diff
@@ -525,7 +525,7 @@ def filename(self):
     description = models.TextField(blank=True, null=True, default=None)
     eo_gsd = models.FloatField(null=True, blank=True, validators=[validate_eo_gsd])
 
-    class Language(models.TextChoices):
+    class Language(models.TextChoices):  # pylint: disable=too-many-ancestors
```

Comment:
Would be better to update the pylint configuration to allow more ancestors instead of repeating this pylint disable statement